src/openforms/contrib/customer_interactions/client.py

```diff
@@ -72,15 +81,92 @@ def create_customer_contact(
                 }
             },
         }
+        if party_uuid:
+            data["betrokkene"]["wasPartij"] = {"uuid": party_uuid}
+
         return self.methods.maak_klant_contact(data=data)
 
-    def create_digital_address_for_betrokkene(
-        self, address: str, address_type: SoortDigitaalAdres, betrokkene_uuid: str
-    ):
+    def create_digital_address(
+        self,
+        address: str,
+        address_type: SoortDigitaalAdres,
+        is_preferred: bool,
+        betrokkene_uuid: str,
+        party_uuid: str | None = None,
```

Comment:
I'm not a fan of `str | None` unions, unless the empty string doesn't mean that processing can ignored:


```suggestion
        party_uuid: str = "",
```
simplicity in types usually means simpler code :D

Reply:
changed